### ISSUE TITLE
Why deno install always re-downloads dependencies?

### ISSUE THREAD

**Issue by keroxp** [denoland/deno_std#622]
- https://github.com/denoland/deno_std/blob/master/installer/mod.ts#L230
- `deno install` ignores module caches
- Why?
- It should reload when `--reload` options provided, shouldn't it?
- cc: @bartlomieju

**Comment by bartlomieju** [comment]
Hi @keroxp so I added `--reload` there to ensure that you always install newest version of script.

Example:
Say you want to install `file_server` from `deno_std`, I can install specific version or latest one.
```
// install v0.19.0
$ deno install file_server https://deno.land/std@v0.19.0/http/file_server.ts

// install latest master
$ deno install file_server https://deno.land/std/http/file_server.ts
```
So in case of installing with tag, then sure `--reload` is not needed because scripts won't change with the same version. But what about latest version? If we did not pass `--reload` flag to `deno fetch` and you install it again with exactly same command then `file_server` won't be updated because `deno fetch` sees that this package is already in cache.
